### DISCUSSION TITLE
Generated URLs and IP protection

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#946] [Help] [answered]
Hello. I just want to know what is the recommended way to handle IP protection of my website that is done through CloudFlare and the generated URLs that Vercel uses on every deployment. Aren’t they in a way bypassing this protection? For example I have my custom domain from which you can access the website, however Vercel has also created one similar to: `https://domain-com-live-RANDOM-username.vercel.app/`. I was not able to find a similar question.

**Comment by swarnava** [comment, marked as answer]
Hi there,

We aren't fully aware of how Cloudflare IP protection works behind the scene. I will recommend reaching out to their support team for more information on this :)
